Branch-SDK/src/io/branch/referral/BranchStrongMatchHelper.java

```diff
@@ -144,6 +144,8 @@ private Uri buildStrongMatchUrl(String matchDomain, DeviceInfo deviceInfo, PrefH
             String uriString = "https://" + matchDomain + "/_strong_match?os=" + deviceInfo.getOsName();
             // Add HW ID
             uriString += "&" + Defines.Jsonkey.HardwareID.getKey() + "=" + deviceInfo.getHardwareID();
+            String hardwareIDTypeVal = deviceInfo.isHardwareIDReal() ? Defines.Jsonkey.HardwareIDTypeVendor.getKey() : Defines.Jsonkey.HardwareIDTypeRandom.getKey();
```

Comment:
Won't `HardwareIDTypeRandom` return a different random variable here than when used for `v1/install`?